web/www/horas/Latin/Sancti/12-27.txt

```diff
@@ -225,23 +233,23 @@ $Deo gratias
 Exiit * sermo inter fratres, quod discipulus ille non moritur; et non dixit Jesus, Non moritur: sed, Sic eum volo manere, donec veniam.
 
 [Octava 2]
-!Commemoratio octavae S. Joannis.
-Ant. Iste est Joannes, qui supra pectus Domini in cena recubuit: beatus Apostolus, cui revelata sunt secreta caelestia.
+!Commemoratio Octavae S. Joannis
+Ant. Iste est Joannes, qui supra pectus Domini in caena recubuit: beatus Apostolus, cui revelata sunt secreta coelestia.
```

Comment:
I think it would be difficult to use different orthographic styles in different sections of the same file. At the very least, if this is to be done, it will be necessary to document the style to follow in each section.

However, if there is going to be code that changes the orthographic style, then there is no need to use different styles in different parts of the database of texts: all texts can be kept in a single style, and then converted to the style chosen for output.

Another consideration is that I doubt that 1902 edition was an *editio typica.* That means that there could have been other editions at the same time following different orthographic styles, based on the preferences of their publishers.